tools/tidb-binlog-cluster.md

```diff
@@ -132,9 +132,33 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
         pump3 ansible_host=172.16.10.74 deploy_dir=/data1/pump
         ```
 
-2. 部署并启动 TiDB 集群
+2. 部署并启动含 Pump 组件的 TiDB 集群
+
+    参照上文配置完 `inventory.ini` 文件后，从以下两种方式中选择一种进行部署。
+    
+    **方式一**：在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行。
+    
+    1. 部署 pump_servers 和 node_exporters
+    
+        ```
+        ansible-playbook deploy.yml -l pump_servers,monitored_servers
+        ```
+        
+    2. 更新并重启 tidb_servers
+    
+        ```
+        ansible-playbook rolling_update.yml --tags=tidb
+        ```
+        
+    3. 更新监控信息
+    
+        ```
+        ansible-playbook rolling_update_monitor.yml
```

Comment:
Just update the prometheus configuration: `ansible-playbook rolling_update_monitor.yml --tags=prometheus`